Tests/AK/TestNumberFormat.cpp

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "LibTest/Macros.h"
```

Comment:
This include isn't required, looks like a clang-tidy addition (if it were required it should be in `<>` not `""`